### ISSUE TITLE
Change template should  not allow to change app

### ISSUE THREAD

**Issue by eXistenZe** [2sic/2sxc#1326]
**I'm submitting a ...** 
[x] feature request

**...about** 
[x] edit experience / UI
[x] admin experience UI

**Current behavior**
If you click "change template", and then the app name on the left, you are allowed to click on any app. If you do, it will show the progress bar, but will never finish and when you reload it will show the error on the screenshot. https://pasteboard.co/GNPJ6d7.jpg

**Expected behavior**
Change template should only show the templates within the app.

**Comment by iJungleboy** [comment]
You are right. Can you temporarily fix it, by re-selecting the original app?

**Comment by eXistenZe** [comment]
Yes. It hangs again, but after reload it works fine.
